sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -251,47 +247,43 @@ public void setExecutorVolume(VolumeSpec volumeSpec) {
-                            .setOutputFile(getConfigTemplateDownloadPath(config))
-                            .setExtract(false);
-                }
+            // Any URIs defined in PodSpec itself.
+            for (URI uri : podSpec.getUris()) {
+                commandBuilder.addUrisBuilder().setValue(uri.toString());
+            }
 
-                // Secrets are constructed differently from other envvars where the proto is concerned:
-                for (SecretSpec secretSpec : podInstance.getPod().getSecrets()) {
-                    if (secretSpec.getEnvKey().isPresent()) {
-                        commandBuilder.getEnvironmentBuilder().addVariablesBuilder()
-                                .setName(secretSpec.getEnvKey().get())
-                                .setType(Protos.Environment.Variable.Type.SECRET)
-                                .setSecret(getReferenceSecret(secretSpec.getSecretPath()));
-                    }
-                }
+            for (ConfigFileSpec config : taskSpec.getConfigFiles()) {
+                commandBuilder.addUrisBuilder()
+                        .setValue(templateUrlFactory.get(
+                                targetConfigurationId, podSpec.getType(), taskSpec.getName(), config.getName()))
+                        .setOutputFile(getConfigTemplateDownloadPath(config))
+                        .setExtract(false);
+            }
 
-                if (podSpec.getUser().isPresent()) {
-                    commandBuilder.setUser(podSpec.getUser().get());
+            // Secrets are constructed differently from other envvars where the proto is concerned:
+            for (SecretSpec secretSpec : podInstance.getPod().getSecrets()) {
+                if (secretSpec.getEnvKey().isPresent()) {
+                    commandBuilder.getEnvironmentBuilder().addVariablesBuilder()
+                            .setName(secretSpec.getEnvKey().get())
+                            .setType(Protos.Environment.Variable.Type.SECRET)
+                            .setSecret(getReferenceSecret(secretSpec.getSecretPath()));
                 }
             }
+
+            if (podSpec.getUser().isPresent()) {
+                commandBuilder.setUser(podSpec.getUser().get());
+            }
+
         }
 
         if (taskSpec.getDiscovery().isPresent()) {
             taskInfoBuilder.setDiscovery(getDiscoveryInfo(taskSpec.getDiscovery().get(), podInstance.getIndex()));
         }
 
-        if (useDefaultExecutor) {
-            taskInfoBuilder.setContainer(getContainerInfo(podInstance.getPod(), true, true));
-        } else if (!podInstance.getPod().getNetworks().isEmpty()) {
-            taskInfoBuilder.setContainer(getContainerInfo(podInstance.getPod(), false, true));
-        } else {
-            taskInfoBuilder.setContainer(Protos.ContainerInfo.newBuilder().setType(Protos.ContainerInfo.Type.MESOS));
-        }
+
+        taskInfoBuilder.setContainer(getContainerInfo(podInstance.getPod(), true, true));
```

Comment:
Are either of the boolean parameters to getContainerInfo always true with this change?